docs/configuration.md

```diff
@@ -130,6 +130,27 @@ fladle {
     defaultClassTestTime = 180.5
     useAverageTestTimeForNewTests = true
     disableResultsUpload = true
+    grantPermissions = "none"
+    type = "game-loop"
+    scenarioLabels = [
+      "label1",
+      "label2" 
+    ]
+    scenarioNumbers = [ 1, 23, 52 ]
+    obbFiles = [
+      "local/file/path/test1.obb",
+      "local/file/path/test2.obb"
+    ]
+    obbNames = [
+      "patch.0300110.com.example.android.obb",
+      "patch.0300111.com.example.android.obb"
+    ]
+    testTargetsForShard = [
```

Comment:
nice, thanks for adding this. however, i think it is missing in the kotlin sample.

Reply:
Ignore this comment, the sample configuration has only a groovy version.